dk-installer.py

```diff
@@ -335,8 +344,15 @@ class Action:
 
     @contextlib.contextmanager
     def init_session_folder(self, prefix):
-        script_path = pathlib.Path(sys.argv[0]).absolute()
-        data_folder = script_path.parent.joinpath(".dk-installer")
+        if detect_os() == 'Windows':
+            home_dir = pathlib.Path.home()
+            data_folder = home_dir.joinpath("Documents", "dk-installer", "logs")
+            if not data_folder.exists():
+                data_folder.mkdir(parents=True, exist_ok=True)
```

Comment:
You don't have to check if the folder exists, because you are calling `mkdir` with `ok_exists=True`. Also, this folder is being created anyway just a few lines below, so you don't even need these two lines

```suggestion
```